docs/Doxyfile.in

```diff
@@ -1118,15 +1110,15 @@ VERBATIM_HEADERS       = YES
 # generated with the -Duse_libclang=ON option for CMake.
 # The default value is: NO.
 
-CLANG_ASSISTED_PARSING = NO
+# CLANG_ASSISTED_PARSING = NO
```

Comment:
do the changes in this file belong to this PR?

Reply:
looking at master id does not look like. i think chanegs in Doxyfile.in should be reverted

Reply:
removed all doxygen file changes